debugger/pythonExtension/signal_handler.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+import gdb
+import sys
+
+def oe_stop_event_handler(event):
+    if isinstance(event, gdb.SignalEvent) and event.stop_signal == "SIGILL":
+        try:
+            frame = gdb.newest_frame()
+            # Disassemble to find out the instructio
+            # See: https://wiki.osdev.org/X86-64_Instruction_Encoding
+            max_instruction_length = 15
+            start_pc = frame.pc()
+            end_pc = start_pc + max_instruction_length
+            asm = frame.architecture().disassemble(start_pc= start_pc,
+                                                   end_pc = end_pc)
+
+            if asm[0]['asm'].startswith('cpuid'):
+                # Continue execution
+                print("oegdb: Ignoring emulated instruction cpuid.")
+                gdb.execute("continue", from_tty=True, to_string=False)
+        except:
+            print("Unexpected error:", sys.exc_info()[1])
+
+def register():
```

Comment:
Minor suggestion: `signal_handler.register` seems like it would take as a parameter the signal handler. Maybe `signal_handler.register_default()`